### ISSUE TITLE
Actuator not updated

### ISSUE THREAD

**Issue by cdupont** [Waziup/WaziGate#124]
Today I tried the actuation. I created the actuator on the WaziGate and it was created correctly on the WaziCloud.
However, after changing the actuation value, it was not updated on the WaziGate.

![Peek 2020-12-03 17-21](https://user-images.githubusercontent.com/484033/101057139-0295d700-358c-11eb-95cf-dab3548ec993.gif)
Traces:
```
2020-12-03T16:20:55.946845530Z [DB   ] Actuator 5fc8f1500ab07000064d546a/5fc910670ab07000064d564f created.
2020-12-03T16:20:55.946925584Z [UP   ] Status "/devices/5fc8f1500ab07000064d546a/actuators/5fc910670ab07000064d564f": create
2020-12-03T16:20:55.947744555Z [HTTP ] 10.42.0.1:42740 200 POST "/devices/5fc8f1500ab07000064d546a/actuators" s:133
2020-12-03T16:20:55.947815758Z [UP   ] Wakeup on flagged entity.
2020-12-03T16:20:55.947883590Z [UP   ] Pushing actuator 5fc8f1500ab07000064d546a/5fc910670ab07000064d564f ...
2020-12-03T16:20:56.005027078Z [UP   ] Status "/devices/5fc8f1500ab07000064d546a/actuators/5fc910670ab07000064d564f": released
2020-12-03T16:20:56.005094466Z [UP   ] The cloud will sleep for 59m59.999989149s ... zZzZ
2020-12-03T16:21:01.930252048Z [MQTT ] "chirpstack-application-s" Message "gateway/0242ac120003ccde/event/up" s:100 r:false q:0
2020-12-03T16:21:02.026405762Z [HTTP ] 10.42.0.1:42740 200 GET "/sys/uptime" s:0
2020-12-03T16:21:02.151850812Z [MQTT ] "chirpstack-application-s" Message "application/55/device/aa555a002601143f/rx" s:395 r:false q:0
2020-12-03T16:21:02.155761928Z [DB   ] 1 value for 5fc8f1500ab07000064d546a/temperatureSensor_1.
2020-12-03T16:21:02.155827279Z [UP   ] Status "/devices/5fc8f1500ab07000064d546a/sensors/temperatureSensor_1": sync
2020-12-03T16:21:02.156077831Z [HTTP ] 172.18.0.4:41044 200 POST "/devices/5fc8f1500ab07000064d546a/sensors/temperatureSensor_1/value" s:2
2020-12-03T16:21:02.156136700Z [UP   ] Wakeup on flagged entity.
2020-12-03T16:21:02.248718628Z [MQTT ] "chirpstack-network-serve" Message "gateway/0242ac120003ccde/command/down" s:86 r:false q:0
2020-12-03T16:21:02.251858550Z [MQTT ] "chirpstack-application-s" Message "gateway/0242ac120003ccde/event/ack" s:32 r:false q:0
2020-12-03T16:21:03.434833052Z [UP   ] Pushed 1 values until 2020-12-03 16:21:02 +0000 UTC, buffered -153.776287ms.
2020-12-03T16:21:03.434918033Z [UP   ] Status "/devices/5fc8f1500ab07000064d546a/sensors/temperatureSensor_1": sync
2020-12-03T16:21:03.435097401Z [UP   ] The cloud will sleep for 4.999790669s ... zZzZ
2020-12-03T16:21:06.169090018Z [MQTT ] "chirpstack-application-s" Message "gateway/0242ac120003ccde/event/stats" s:56 r:false q:0
2020-12-03T16:21:08.305655897Z [WWW  ] (127.0.0.1:44038) 200 GET "/"
2020-12-03T16:21:08.435158764Z [UP   ] Wakeup on timer.
2020-12-03T16:21:08.436004012Z [UP   ] Status "/devices/5fc8f1500ab07000064d546a/sensors/temperatureSensor_1": released
2020-12-03T16:21:08.436172473Z [UP   ] Values for this entity are now up-to-date.
2020-12-03T16:21:08.436205454Z [UP   ] The cloud will sleep for 59m59.999985315s ... zZzZ
2020-12-03T16:21:10.737370645Z [HTTP ] 10.42.0.1:42740 200 POST "/auth/retoken" s:2
2020-12-03T16:21:10.767878521Z [HTTP ] 10.42.0.1:42740 200 GET "/sys/uptime" s:0
2020-12-03T16:21:14.295010276Z [MQTT ] "dashboard_4cbda3d3" Subscribed "devices" qos:0
2020-12-03T16:21:14.295817136Z [HTTP ] 10.42.0.1:42740 200 GET "/device/id" s:0
2020-12-03T16:21:14.296821086Z [HTTP ] 10.42.0.1:43142 200 GET "/devices" s:0
2020-12-03T16:21:17.128432025Z [HTTP ] 10.42.0.1:42740 200 GET "/sys/uptime" s:0
2020-12-03T16:21:18.458727862Z [MQTT ] "chirpstack-application-s" Message "gateway/0242ac120003ccde/event/up" s:100 r:false q:0
2020-12-03T16:21:18.701509403Z [MQTT ] "chirpstack-application-s" Message "application/55/device/aa555a002601143f/rx" s:395 r:false q:0
2020-12-03T16:21:18.708748956Z [DB   ] 1 value for 5fc8f1500ab07000064d546a/temperatureSensor_1.
2020-12-03T16:21:18.708868862Z [UP   ] Status "/devices/5fc8f1500ab07000064d546a/sensors/temperatureSensor_1": sync
2020-12-03T16:21:18.708940046Z [HTTP ] 172.18.0.4:41044 200 POST "/devices/5fc8f1500ab07000064d546a/sensors/temperatureSensor_1/value" s:2
2020-12-03T16:21:18.709545353Z [UP   ] Wakeup on flagged entity.
2020-12-03T16:21:18.792618240Z [MQTT ] "chirpstack-network-serve" Message "gateway/0242ac120003ccde/command/down" s:86 r:false q:0
2020-12-03T16:21:18.796850554Z [MQTT ] "chirpstack-application-s" Message "gateway/0242ac120003ccde/event/ack" s:32 r:false q:0
2020-12-03T16:21:18.825104886Z [UP   ] Pushed 1 values until 2020-12-03 16:21:18 +0000 UTC, buffered -703.710171ms.
2020-12-03T16:21:18.825226014Z [UP   ] Status "/devices/5fc8f1500ab07000064d546a/sensors/temperatureSensor_1": sync
2020-12-03T16:21:18.825357290Z [UP   ] The cloud will sleep for 4.999774466s ... zZzZ
2020-12-03T16:21:23.825924049Z [UP   ] Wakeup on timer.
2020-12-03T16:21:23.826785982Z [UP   ] Status "/devices/5fc8f1500ab07000064d546a/sensors/temperatureSensor_1": released
2020-12-03T16:21:23.826835185Z [UP   ] Values for this entity are now up-to-date.
2020-12-03T16:21:23.826880777Z [UP   ] The cloud will sleep for 59m59.999989203s ... zZzZ
2020-12-03T16:21:26.758865799Z [HTTP ] 10.42.0.1:42740 200 GET "/sys/uptime" s:0
2020-12-03T16:21:32.227174761Z [HTTP ] 10.42.0.1:42740 200 GET "/sys/uptime" s:0
2020-12-03T16:21:34.965131876Z [MQTT ] "chirpstack-application-s" Message "gateway/0242ac120003ccde/event/up" s:100 r:false q:0
2020-12-03T16:21:35.184255006Z [MQTT ] "chirpstack-application-s" Message "application/55/device/aa555a002601143f/rx" s:395 r:false q:0
2020-12-03T16:21:35.187723757Z [DB   ] 1 value for 5fc8f1500ab07000064d546a/temperatureSensor_1.
2020-12-03T16:21:35.187794960Z [UP   ] Status "/devices/5fc8f1500ab07000064d546a/sensors/temperatureSensor_1": sync

```

**Comment by j-forster** [comment]
This has been fixed by https://github.com/Waziup/wazigate-edge/commit/a3a99bcf13468b97de966548709c3006fa36bc58.

**Comment by cdupont** [comment]
Fixed in ISO 2.1.3, closing
